packages/eslint-config-4catalyzer-react/rules.js

```diff
@@ -61,6 +50,7 @@ module.exports = {
         'getters',
         'setters',
         '/^(get|set)(?!(InitialState$|DefaultProps$|ChildContext$)).+$/',
+        'instance-methods',
```

Comment:
why not

Reply:
does this make you need to group arrow functions and methods? That was one of the motivating reasons to override this.

Reply:
i thought we overrode this to insert the thing for type definitions

Reply:
also that